CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file.
 
+## \[v0.5.0\]
+
+### Added
+
+- Model Preparation Algorithm (MPA)
+  - Add tiling in rotated detection (<https://github.com/openvinotoolkit/training_extensions/pull/1420>)
+  - Add Cythonize AugMixAugment (<https://github.com/openvinotoolkit/training_extensions/pull/1478/>)
+- Integrate ov-telemetr (<https://github.com/openvinotoolkit/training_extensions/pull/1568>)
```

Comment:
A simple typo? `ov-telemetr` -> `ov-telemetry`

Reply:
> A simple typo? `ov-telemetr` -> `ov-telemetry`
Sorry. I fixed it. Thanks